internal/ingress/controller/config/config.go

```diff
@@ -768,19 +768,7 @@ func NewDefault() Configuration {
 	defNginxStatusIpv6Whitelist := make([]string, 0)
 	defResponseHeaders := make([]string, 0)
 
-	defAnnotationValueWordBlocklist := []string{
-		"load_module",
-		"lua_package",
-		"_by_lua",
-		"location",
-		"root",
-		"proxy_pass",
-		"serviceaccount",
-		"{",
-		"}",
-		"'",
-		"\\",
-	}
+	defAnnotationValueWordBlocklist := make([]string, 0)
```

Comment:
You can remove this line :)

Then in (on the defaults definitions)
```
AnnotationValueWordBlocklist: strings.Join(defAnnotationValueWordBlocklist, ","),
```
Replace with an empty string:
```
AnnotationValueWordBlocklist: "",
```